docs/usage/configuration-options.md

```diff
@@ -1695,7 +1695,7 @@ If you wish to enable this feature then you could add this to your configuration
 }
```

Comment:
The tests are okay after my changes. I expected things to break on me somehow.

It still feels too much like a "draft" for me. It was more of a copy terms into IDE and find/replace manually what seems right to change...

Please thoroughly review this PR before you mark it ready to merge.